scripts/completion/dvc.bash

```diff
@@ -4,63 +4,75 @@
 #   - https://opensource.com/article/18/3/creating-bash-completion-script
 #   - https://stackoverflow.com/questions/12933362
 
-_dvc_commands='add cache checkout commit config destroy diff fetch freeze get-url get gc \
-  import-url import init install list metrics move params pipeline plots pull push \
-  remote remove repro root run status unfreeze unprotect update version'
+_dvc_commands='add cache checkout commit config daemon destroy diff fetch freeze gc get get-url git-hook import import-url init install list metrics move params pipeline plots pull push remote remove repro root run status unfreeze unprotect update version'
 
 _dvc_options='-h --help -V --version'
 _dvc_global_options='-h --help -q --quiet -v --verbose'
 
-_dvc_add='-R --recursive -f --file --no-commit'
+_dvc_add='-R --recursive --no-commit -f --file'
 _dvc_add_COMPGEN=_dvc_compgen_files
 _dvc_cache='dir'
-_dvc_cache_dir=' --global --system --local -u --unset'
-_dvc_checkout='-d --with-deps -R --recursive -f --force --relink --summary'
+_dvc_cache_dir='--global --system --local -u --unset'
+_dvc_cache_dir_COMPGEN=_dvc_compgen_files
+_dvc_checkout='--summary -d --with-deps -R --recursive -f --force --relink'
 _dvc_checkout_COMPGEN=_dvc_compgen_DVCFiles
 _dvc_commit='-f --force -d --with-deps -R --recursive'
 _dvc_commit_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_config='-u --unset --local --system --global'
+_dvc_config='--global --system --local -u --unset'
+_dvc_daemon='updater analytics'
+_dvc_daemon_analytics=''
+_dvc_daemon_analytics_COMPGEN=_dvc_compgen_DVCFiles #?
```

Comment:
need to check this line

Reply:
@casperdcl `dvc daemon` is more of a service command, it is not meant to be ran by users. So we can safely skip it. But it accepts just files, not dvcfiles.